pkg/image/cdi/common.go

```diff
@@ -93,6 +92,16 @@ func generateDVSourceHTTP(vmi *harvesterv1.VirtualMachineImage) *cdiv1.DataVolum
 	}
 }
 
+func GenerateDVAnnotations(sc *storagev1.StorageClass) map[string]string {
+	annotations := map[string]string{}
+	volBindingMode := *sc.VolumeBindingMode
```

Comment:
need to check if VolumeBindingMode is not nil first
```
	// volumeBindingMode indicates how PersistentVolumeClaims should be
	// provisioned and bound.  When unset, VolumeBindingImmediate is used.
	// This field is only honored by servers that enable the VolumeScheduling feature.
	// +optional
	VolumeBindingMode *VolumeBindingMode `json:"volumeBindingMode,omitempty" protobuf:"bytes,7,opt,name=volumeBindingMode"`
```

Reply:
updated. The default volumeBindingMode is immediate. We could do nothing about this.